rpcs3/Emu/RSX/RSXFragmentProgram.h

```diff
@@ -265,7 +266,8 @@ struct RSXFragmentProgram
 				data_ptr = other.data_ptr;
 				local_storage.clear();
 			}
-
+			
```

Comment:
what is this line change?

Reply:
i think it's just auto indenting whitespace from the ide :*